src/features/ShowOnlineHelp.ts

```diff
@@ -15,20 +15,24 @@ export class ShowHelpFeature implements IFeature {
     private languageClient: LanguageClient;
 
     constructor() {
-        this.command = vscode.commands.registerCommand("PowerShell.OnlineHelp", () => {
+        this.command = vscode.commands.registerCommand("PowerShell.OnlineHelp", (item?) => {
             if (this.languageClient === undefined) {
                 // TODO: Log error message
                 return;
             }
+            if (item === undefined) {
 
-            const editor = vscode.window.activeTextEditor;
+                const editor = vscode.window.activeTextEditor;
 
-            const selection = editor.selection;
-            const doc = editor.document;
-            const cwr = doc.getWordRangeAtPosition(selection.active);
-            const text = doc.getText(cwr);
+                const selection = editor.selection;
+                const doc = editor.document;
+                const cwr = doc.getWordRangeAtPosition(selection.active);
+                const text = doc.getText(cwr);
 
-            this.languageClient.sendRequest(ShowOnlineHelpRequestType, text);
+                this.languageClient.sendRequest(ShowOnlineHelpRequestType, text);
+            } else {
+                this.languageClient.sendRequest(ShowOnlineHelpRequestType, item.Name);
```

Comment:
Is this a part of the PSES PR that adds the extra logic for OnlineHelp?

Reply:
No, I don't believe these conflict as this is just adding logic to what we send on if they're selecting the text or not. Based on your suggestion here: https://github.com/PowerShell/PowerShellEditorServices/pull/697#discussion_r213069053 I would be more inclined to change these too PRs back to WIP and refactor that to a single message call.

Reply:
These only conflicted this time around because we renamed the request 😄